src/sentry/integrations/vercel/client.py

```diff
@@ -41,40 +58,35 @@ def request(self, method, path, data=None, params=None, allow_text=False):
 
     def get_team(self):
         assert self.team_id
-        return self.get(self.TEAMS_URL % self.team_id)
+        return self.get(self.GET_TEAM_URL % self.team_id)
 
     def get_user(self):
-        return self.get(self.USER_URL)["user"]
+        return self.get(self.GET_USER_URL)["user"]
 
-    def paginate(self, url, type):
-        limit = 20
+    def get_from_pagination(self, url, data_key):
+        # Vercel Pagination Guide: https://vercel.com/docs/rest-api#introduction/api-basics/pagination
+        limit = 100
         params = {"limit": limit}
         results = []
-        # no one should have more than 200 results
-        for i in range(10):
-            resp = self.get(url, params=params)
-            results += resp[type]
-            # if we have less results than the limit, we are done
-            if resp["pagination"]["count"] < limit:
-                return results
-            # continue pagination by setting the until parameter
-            params = params.copy()
-            params["until"] = resp["pagination"]["next"]
-        # log the warning if this happens so we can look into solutions
-        logger.warning("Did not finish pagination", extra={"team_id": self.team_id, "url": url})
+        next_timestamp = ""
+        while next_timestamp is not None:
```

Comment:
This does have the possibility of getting stuck making these requests looping through projects, but we previously assumed they would have less than 200 (and even that never got met). 

Raising the limit to 100, means this would likely iterate 2 or fewer times in practice